lldb/source/Core/Module.cpp

```diff
@@ -1538,6 +1538,9 @@ bool Module::MatchesModuleSpec(const ModuleSpec &module_ref) {
   if (!FileSpec::Match(platform_file_spec, GetPlatformFileSpec()))
     return false;
 
+  if (m_object_offset != module_ref.GetObjectOffset())
```

Comment:
This change has been moved into a separate PR: https://github.com/clayborg/llvm-project/pull/5